### ISSUE TITLE
Milestone 5 Task 2: RET API access

### ISSUE THREAD

**Issue by FLX-20** [FLX-20/Data-Science-Toolkits-and-Architectures-group-1#61]
Create a Flask application that allows the following:
-  Expose a REST endpoint (for example <base_url>/predict)
 - This endpoint should accept a POST Request
 - The body of the POST Request should contain an Image out of your dataset (MNIST)
 
You can send an image to this REST endpoint using the requests library; example:
 (https://stackoverflow.com/questions/60636701/how-to-send-post-request-with-base64-image)

 The Flask app should accept the image from this endpoint and decode it such that you get a numpy
 array. Then provide the array to your neural network. Neural network prediction and image should be
 saved in the database (as was done in Milestone 3).  

 The Flask app should return the prediction to the client (calling the REST API)
 In other words: use the requests Python library to send a POST request containing an image with
 handwritten digits to a REST endpoint provided by Flask, which runs the neural network and returns it’s
 prediction to the caller.

 Note that all parts of this system have to be dockerized (Flask app and database) and can be started with a
 simple “docker-compose up”
